src/Playwright/build/playwright.ps1

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env pwsh
 
-$PlaywrightFileName = Join-Path $PSScriptRoot "Microsoft.Playwright.dll"
-[Reflection.Assembly]::LoadFile($PlaywrightFileName) | Out-Null
+$Env:PLAYWRIGHT_DRIVER_SEARCH_PATH = $PSScriptRoot;
+$playwrightLibrary = [System.IO.Path]::GetFullPath([System.IO.Path]::Combine($PSScriptRoot, "Microsoft.Playwright.dll"))
+[Reflection.Assembly]::Load([System.IO.File]::ReadAllBytes($playwrightLibrary)) | Out-Null
```

Comment:
This doesn't seem right given executable location information is lost. Maybe there is a way to make dll unload after the executable finishes?